hilbert/src/main/scala/org/locationtech/sfcurve/hilbert/HilbertCurve2D.scala

```diff
@@ -88,7 +88,7 @@ class HilbertCurve2D(resolution: Int) extends SpaceFillingCurve2D {
     (x, y)
   }
 
-  def toRanges(xmin: Double, ymin: Double, xmax: Double, ymax: Double): Seq[(Long, Long)] = {
+  def toRanges(xmin: Double, ymin: Double, xmax: Double, ymax: Double, maxRecurse: Int = -1 /* not used */): Seq[(Long, Long, Boolean)] = {
```

Comment:
What is the max recurse here for, if it's unused? Is this an implementation-specific parameter leaking into the interface?